### ISSUE TITLE
You

### ISSUE THREAD

**Issue by kreezxil** [lukesUbuntu/ns-minecraft-panel#1]
# gave up didn't you?

**Comment by lukesUbuntu** [comment]
i have another minecraft panel under a private repo as i went with another stack. I will tidy it up and release it as public. The stack i moved to was Lumen + Angular 6

**Comment by kreezxil** [comment]
is it something I can install alongside an existing ubuntu installation
such that it could be run like an app and not require a fresh install of
ubuntu to be built upon? I see that alot these interfaces wanting to be the
only thing on a server not playing nice with other things that might
already be there.

**Comment by lukesUbuntu** [comment]
yes i will add in a setup wizard so it checks the current setup ill try get it done over the weekend trying to make it as easy as possible to install and setup

**Comment by kreezxil** [comment]
sweet.
